content/docs/atom-guide/look-dev/materials/material-system.md

```diff
@@ -0,0 +1,80 @@
+---
+linktitle: System Overview
+title: "Material System Overview"
+description: "Materials contain data that control how model surfaces appear in a 3D environment."
+toc: false
```

Comment:
Setting toc to false does not currently work. And I'm not sure we'd want it here if it did.
```suggestion
```